### DISCUSSION TITLE
where do i run the setup function?

### DISCUSSION THREAD

**Discussion by Thebuilderekes** [lukas-reineke/indent-blankline.nvim#784] [General]
Hi, I'm new to this, I've installed the plugin using vimplug into my config file, which is a vimscript based config file and was wondering where to run the setup function ` require("ibl").setup()`  to get it working?

**Comment by lukas-reineke** [comment]
Anywhere after you initiallize vimplug. And you can use `:help :lua-heredoc` to do it from vimscipt.
